src/utilities/getCurrentDate.ts

```diff
@@ -57,10 +59,12 @@ export function generateDateObject(day: Date): DateData {
 	return DateObject;
 }
 
-export function generateDateOrig(currentDate: DateData): Date {
-	const month = currentDate.month;
-	const day = currentDate.day;
-	const year = currentDate.year;
+export function generateDateOrig(currentDate: DateData) {
+	const currentDateCopy = { ...currentDate };
+
+	const month = getMonthString(currentDateCopy.month);
```

Comment:
Originally set month to number. Which would generate the proper date in browser JS. But mobile/device caused an error where NaN would generate. Changed month to string so mobile devices can generate the new Date properly.